tests/schema/tests.py

```diff
@@ -215,13 +228,7 @@ def test_fk(self):
         with connection.schema_editor() as editor:
             editor.alter_field(Book, old_field, new_field, strict=True)
         # Make sure the new FK constraint is present
```

Comment:
Now that the assertion has a good name, I would remove these redundant comments.